src/api/data-structure-creator.js

```diff
@@ -0,0 +1,100 @@
+export class DataStructureCreate {
+    constructor(name) {
+        if (!name) {
+            throw new Error('name is required.');
+        } else if (_isNotString(name)) {
+            throw new Error('name should be string');
+        } else {
+            this.name = name;
+        }
+        this.items = JSON.parse(localStorage.getItem(`${name}`) || "[]");
```

Comment:
If `name` is already a string you don't need to wrap it in a string literal
```suggestion
        this.items = JSON.parse(localStorage.getItem(name) || "[]");
```